Equipment/EquipmentDb/Shoulders/leather.xml

```diff
@@ -206,4 +206,34 @@
 		Crafted by Leatherworkers.
 		</source>
 	</item>
+	
+	<item id="20194" phase="4">
+		<info name="Defiler's Leather Shoulders" type="LEATHER" slot="SHOULDERS" unique="yes" req_lvl="60" item_lvl="65" quality="EPIC" boe="no" icon="Inv_shoulder_24.png"/>
```

Comment:
There is a `faction` attribute to properly hide/show depending on current faction, but I'll add this myself as you have done far and away enough work on this pull request. :-)